addons/medical_statemachine/functions/fnc_leftStateCardiacArrest.sqf

```diff
@@ -1,7 +1,8 @@
 #include "script_component.hpp"
 /*
- * Author: BaerMitUmlaut
- * Handles a unit entering cardiac arrest.
+ * Author: RedBery
```

Comment:
Author: BaerMitUmlaut, RedBery

Reply:
Baer's name was only here because the enteredStateCardiacArrest file was copied over and edited so the header was for that function.

Reply:
ok. I got confused :)